presto-parser/src/main/java/com/facebook/presto/sql/tree/QuantifiedComparisonExpression.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+
+public class QuantifiedComparisonExpression
+        extends Expression
+{
+    public enum Quantifier
+    {
+        ALL,
+        ANY,
+        SOME,
```

Comment:
Is there any semantic difference between `ANY` and `SOME`? If there is not, the I would have only one of them (`ANY`) and just map in `AstBuilder` `SOME` to `ANY`.